server/data/interfaces/prisonerSearchApi/Prisoner.ts

```diff
@@ -120,7 +120,6 @@ export default interface Prisoner {
   tattoos?: BodyPartDetail[]
   scars?: BodyPartDetail[]
   marks?: BodyPartDetail[]
-  otherMarks?: BodyPartDetail[]
```

Comment:
Removing from response model since prisoner search no longer provides this - it wasn't used anyway